yb-voyager/src/metadb/migrationStatus.go

```diff
@@ -15,15 +17,27 @@ type MigrationStatusRecord struct {
 	TargetDBConf                *tgtdb.TargetConf
 	FallForwardDBConf           *tgtdb.TargetConf
 	TableListExportedFromSource []string
+	Triggers                    []string
+	MigInfo                     *MigInfo
+}
+
+type MigInfo struct {
+	SourceDBType    string
+	SourceDBName    string
+	SourceDBSchema  string
+	SourceDBVersion string
+	SourceDBSid     string
+	SourceTNSAlias  string
+	ExportDir       string
```

Comment:
We already dump the `tgtdb.TargetConf` (after clearing password fields in it) in the MSR. Should we simply dump the `srcdb.Source` as well and then get rid of `MigInfo`?